ultravox/training/config_base.py

```diff
@@ -54,7 +55,7 @@ class TrainConfig:
     optimizer: str = "adamw_torch"
     num_epochs: int = 1
     max_steps: int = 0
-    eval_steps: Optional[int] = None
+    val_steps: Optional[int] = None
```

Comment:
just checking, did you mean to replace `eval_steps` (as `eval_num_samples` wasn't replaced)

Reply:
No. The idea is that we have 3 things now: training, validation, final (generative) evaluation.

HF calls the second one `eval` (e.g. `eval_dataset`, `eval_steps`), but I wanted to make sure the names `val` refers to validation and `eval` refers to the final generative evaluation.

On that note, I see that I need to rename `eval_accum_steps` to `val_accum_steps` as well to keep things consistent.